flows/actions/classify_nlu.go

```diff
@@ -0,0 +1,130 @@
+package actions
+
+import (
+	"encoding/json"
+
+	"github.com/nyaruka/goflow/assets"
+	"github.com/nyaruka/goflow/flows"
+	"github.com/nyaruka/goflow/flows/events"
+	"github.com/pkg/errors"
+)
+
+func init() {
+	registerType(TypeClassifyNLU, func() flows.Action { return &ClassifyNLUAction{} })
+}
+
+const (
+	classificationCategorySuccess = "success"
+	classificationCategorySkipped = "skipped"
```

Comment:
this for cases like the input evaluating to empty.. no there's nothing to classify.. but it doesn't quite feel like a failure.. cuz we reserve that for actual problems vs flow design issues.

Reply:
Sorry, just more over-communication on this. This is because its a runtime issue, not a flow design issue correct? Flow design issues don't deserve routing structures.

Reply:
Yeah I think it's useful to distinguish between something like "user wants to classify empty input" (not necessarily an error from their pov.. just empty input = empty entities) vs "their credentials for wit.ai are wrong".